arangodb-net-standard.Test/CursorApi/CursorApiClientTest.cs

```diff
@@ -40,6 +65,30 @@ public async Task PostCursorAsync_ShouldSucceed()
             Assert.Equal("This is a robbery", result.First().MyProperty);
         }
 
+        [Fact]
+        public async Task PostCursorAsync_ShouldSucceed_WhenUsingDefaultSerializationOptions()
```

Comment:
Maybe add a similar test for the transaction endpoint?

Reply:
Not sure if I have time right away, is it crucial do you think?

Reply:
Not crucial but a nice to have I think.